CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ x.x.x Release notes (yyyy-MM-dd)
 ### Bugfixes
 
 * Fix CocoaPods installation with static libraries and multiple platforms.
+* Fix uncaught "Bad version number" exceptions when refreshing a Realm with no
+  notification registered blocks.
```

Comment:
I think these words are in the wrong order, and it's possible (although harder) to hit the problematic case even when using explicit notification blocks, so I'd just say something like "Fix uncaught "Bad version number" exceptions on the notification worker thread followed by deadlocks when Realms refresh".